test/ctr_seccomp.bats

```diff
@@ -37,9 +41,9 @@ function teardown() {
 	echo "$output"
 	[ "$status" -eq 0 ]
 
-	cleanup_ctrs
-	cleanup_pods
-	stop_crio
+	#cleanup_ctrs
```

Comment:
TODO: there seemed to be extra flakes on tests where these functions were called twice. I haven't looked deep into it yet, and am not sure I was correct.